api_tests/lib/ethereum.ts

```diff
@@ -90,20 +94,21 @@ export class EthereumWallet {
         return erc20Balance(this.account, contractAddress);
     }
 
-    public async fund(ether: string) {
+    public async fund(ether: string, confirmation: number = 1) {
         const parityPrivateKey =
             "0x4d5db4107d237df6a3d58ee5f70ae63d73d7658d4026f2eefd2f204c81682cb7";
 
         const weiAmount = ethers.utils.parseEther(ether);
-
         const tx = {
             to: this.address(),
             value: weiAmount.toHexString(),
             gasLimit: 21000,
+            chainId: 17,
```

Comment:
Should we fetch the chainId through `ethers` to make sure it is correct?

Reply:
👍Done